kuma/wiki/middleware.py

```diff
@@ -51,6 +51,17 @@ def process_request(self, request):
 
                 return HttpResponseRedirect(new_path)
 
+            elif request.path_info == u'/docs{}'.format(new_path):
+                # Is this a request for a DocumentZone, but with /docs/ wedged
+                # in the url path between the language code and the zone's url_root?
+                new_path = u'/{}{}'.format(request.LANGUAGE_CODE, new_path)
+                query = request.GET.copy()
+                if 'lang' in query:
+                    query.pop('lang')
```

Comment:
This code path (removing the ``lang`` parameter) is never taken because the [LocaleURLMiddleware](https://github.com/jbradberry/kuma/blob/e02f47af2db2c501c0cd56761b7e224c65c2e39c/kuma/core/middleware.py#L30-L37) redirects for the ``lang=fr`` parameter first. It should be removed.